package.json

```
@@ -3,21 +3,30 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
+    "catalogs:easy": "curl -L https://github.com/i-VRESSE/workflow-builder/raw/main/packages/haddock3_catalog/public/catalog/haddock3.easy.yaml | npx js-yaml > app/catalogs/haddock3.easy.json",
+    "catalogs:expert": "curl -L https://github.com/i-VRESSE/workflow-builder/raw/main/packages/haddock3_catalog/public/catalog/haddock3.expert.yaml | npx js-yaml > app/catalogs/haddock3.expert.json",
+    "catalogs:guru": "curl -L https://github.com/i-VRESSE/workflow-builder/raw/main/packages/haddock3_catalog/public/catalog/haddock3.guru.yaml  | npx js-yaml > app/catalogs/haddock3.guru.json",
+    "catalogs": "npm run catalogs:easy && npm run catalogs:expert && npm run catalogs:guru",
     "dev": "remix dev",
-    "start": "remix-serve build",
-    "typecheck": "tsc",
+    "format": "prettier --write .",
     "generate-client": "openapi-generator-cli generate -g typescript-fetch -i http://localhost:8000/api/openapi.json -o app/bartender-client",
+    "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
+    "start": "remix-serve build",
     "test": "vitest",
```

Running locally the test doesn't return but keeps waiting for file changes. Can we change it like so?
```suggestion
    "test": "vitest run",
```